guides/common/modules/proc_executing-a-remote-job.adoc

```diff
@@ -82,3 +82,17 @@ To execute a remote job with custom parameters, complete the following steps:
 +
 Replace _query_ with the filter expression that defines hosts, for example `"name ~ rex01"`.
 For more information about executing remote commands with hammer, enter `hammer job-template --help` and `hammer job-invocation --help`.
+
+.Switching between Hosts
+
+When you are on a particular host in the {ProjectWebUI}, you can navigate between hosts without leaving the page by using the host switcher.
+Click the ⇄ next to the hostname.
```

Comment:
I think icons are usually bold, so it'd be `*⇄*` I guess. Please check using `git grep`.

Reply:
Addressed :+1: